examples/cull-idle/Cull_service_UI/js/script.js

```diff
@@ -0,0 +1,106 @@
+// Task : Handling js functions for cull UI app
+// Date : 12 August 2016
+// Version : 1.0
+// Author : Vigneshwer
+
+// creating a new socket connection
+var socket = new WebSocket("ws://127.0.0.1:8888/ws");
+
+// socket function to interact with the web server
+socket.onopen = function() {
+	console.log("connected to web socket");
+}
+
+socket.onmessage = function(message) {
+	console.log("Recieving :-" + message.data);
```

Comment:
Looks like we have tab-indentation here as well. The convention in JavaScript is two spaces, though I won't object to four if that's easier for you.